components/ButtonBar/src/MDCButtonBar.m

```diff
@@ -418,10 +418,23 @@ - (void)setItems:(NSArray<UIBarButtonItem *> *)items {
 - (void)setButtonsTitleFont:(UIFont *)font forState:(UIControlState)state {
   [_defaultBuilder setTitleFont:font forState:state];
 
-  for (UIView *viewObj in _buttonViews) {
+  for (NSUInteger ix = 0; ix < [_buttonViews count]; ++ix) {
```

Comment:
Nit/Consistency: Prefer "i" for the loop variable

Reply:
Done.